apps/ledger-live-mobile/src/components/CustomImage/ImageHexProcessor.tsx

```diff
@@ -1,7 +1,7 @@
 import { Flex } from "@ledgerhq/native-ui";
 import React from "react";
 import { WebView } from "react-native-webview";
-import { WebViewErrorEvent, WebViewMessageEvent } from "react-native-webview/lib/WebViewTypes";
+import { WebViewErrorEvent, WebViewMessageEvent } from "react-native-webview/src/WebViewTypes";
```

Comment:
importing from src?

Reply:
👀 Let me double-check, when I tried I had no /lib directory in this library but as I changed a lot of things to make it work on iOS I may don't need this change anymore

Reply:
https://github.com/LedgerHQ/ledger-live/pull/6596#discussion_r1583159043